### PR TITLE
docs(README): Cleanup README and remove outdated installation instructions

### PR DESCRIPTION
**Pull request by MikeRyanDev**
@btroncone Mind reviewing?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,64 +1,53 @@
...
 ```
 
-Install @ngrx/store from npm:
-```bash
-npm install @ngrx/store --save
-```
+Optional packages:
+- [@ngrx/store-devtools](https://github.com/ngrx/store-devtools) instruments your store letting you use a
+[powerful time-travelling debugger](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd?hl=en).
+- [@ngrx/store-router](https://github.com/ngrx/store-router) contains the state of @angular/router in your store
```
Comment (brandonroberts) [inline README.md]: This should be `@ngrx/router-store`

File: README.md [hunk 2]
```diff
@@ -1,64 +1,53 @@
...
-npm install @angular/core @ngrx/core --save
+npm install @ngrx/core @ngrx/store --save
 ```
 
-Install @ngrx/store from npm:
-```bash
-npm install @ngrx/store --save
-```
+Optional packages:
+- [@ngrx/store-devtools](https://github.com/ngrx/store-devtools) instruments your store letting you use a
```
Comment (brandonroberts) [inline README.md:28]: Store devtools could use an example of using the extension also.

File: README.md [hunk 3]
```diff
@@ -1,64 +1,53 @@
 # @ngrx/store
+RxJS powered state management inspired by Redux for Angular 2 applications
```
Comment (brandonroberts) [inline README.md]: Change references from `Angular 2` to just `Angular`

File: README.md [hunk 4]
```diff
@@ -85,24 +74,26 @@ export const counterReducer: ActionReducer<number> = (state: number = 0, action:
 }
 ```
 
-In your app's main module, import those reducers and use the `StoreModule.provideStore(reducers, initialState)` function to provide them to Angular's injector:
+In your app's main module, import those reducers and use the `StoreModule.provideStore(reducers)`
```
Comment (brandonroberts) [inline README.md:77]: Is the example app going to be the source for an AoT example?

File: README.md [hunk 5]
```diff
@@ -85,24 +74,26 @@ export const counterReducer: ActionReducer<number> = (state: number = 0, action:
...
-import { NgModule } from '@angular/core'
 
 @NgModule({
   imports: [
     BrowserModule,
-    StoreModule.provideStore({ counter: counterReducer }, { counter: 0 })
+    StoreModule.provideStore({ counter: counterReducer })
   ]
 })
 export class MyAppModule {}
```
Comment (brandonroberts) [inline README.md]: Change to `AppModule`

File: README.md [hunk 6]
```diff
@@ -85,24 +74,26 @@ export const counterReducer: ActionReducer<number> = (state: number = 0, action:
 }
 ```
 
-In your app's main module, import those reducers and use the `StoreModule.provideStore(reducers, initialState)` function to provide them to Angular's injector:
+In your app's main module, import those reducers and use the `StoreModule.provideStore(reducers)`
+function to provide them to Angular's injector:
 
 ```ts
+import { NgModule } from '@angular/core'
 import { Store, StoreModule } from '@ngrx/store';
```
Comment (brandonroberts) [inline README.md]: Remove `Store` import

File: README.md [hunk 7]
```diff
@@ -85,24 +74,26 @@ export const counterReducer: ActionReducer<number> = (state: number = 0, action:
...
+    StoreModule.provideStore({ counter: counterReducer })
   ]
 })
 export class MyAppModule {}
-
 ```
 
-You can then inject the `Store` service into your components and services. The `store.select` method can be used to obtain the appropriate slice(s) of state from your application store:
+
+You can then inject the `Store` service into your components and services. Use `store.select` to
```
Comment (brandonroberts) [inline README.md:95]: The example needs a reset button

File: README.md (hunk 2 above)
Comment (MikeRyanDev) [reply README.md:28]: Agreed. Will address @ngrx/store-devtools README after this one

File: README.md (hunk 4 above)
Comment (MikeRyanDev) [reply README.md:77]: If we can get AOT compilation working.

File: README.md (hunk 6 above)
Comment (MikeRyanDev) [reply README.md]: Fixed

File: README.md (hunk 7 above)
Comment (MikeRyanDev) [reply README.md:95]: Added a reset button

File: README.md (hunk 3 above)
Comment (MikeRyanDev) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (MikeRyanDev) [reply README.md]: Done

File: README.md (hunk 5 above)
Comment (MikeRyanDev) [reply README.md]: Done

**Comment by btroncone** [comment]
@MikeRyan52 Sure I will review shortly 👍

File: README.md [hunk 8]
```diff
@@ -1,64 +1,53 @@
...
+RxJS powered state management inspired by Redux for Angular applications
 
 [![Join the chat at https://gitter.im/ngrx/store](https://badges.gitter.im/ngrx/store.svg)](https://gitter.im/ngrx/store?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 [![CircleCI Status for ngrx/store](https://circleci.com/gh/ngrx/store.svg?style=shield&circle-token=aea1fc73de3419cd926fc95e627e036113646fd8
 )](https://circleci.com/gh/ngrx/store)
 [![npm version](https://badge.fury.io/js/%40ngrx%2Fstore.svg)](https://badge.fury.io/js/%40ngrx%2Fstore)
 
-RxJS powered state management inspired by Redux for Angular 2 apps
+@ngrx/store is a controlled state container designed to help write performant, consistent applications
+on top of Angular. Essentially:
```
Comment (btroncone) [inline README.md]: I would swap out _Essentially_ with _Core Tenets_.

File: README.md [hunk 9]
```diff
@@ -1,64 +1,53 @@
 # @ngrx/store
+RxJS powered state management inspired by Redux for Angular applications
```
Comment (btroncone) [inline README.md]: Maybe flip this wording to _RxJS powered state management for Angular applications, inspired by Redux._

**Review by btroncone — COMMENTED** [review]
Just had a few wording suggestions, nothing big. Looks great, I really like the changes! 👍

File: README.md (hunk 8 above)
Comment (r-park) [reply README.md]: I think you mean core _tenets_ :)

File: README.md (hunk 9 above)
Comment (MikeRyanDev) [reply README.md]: Changed

File: README.md (hunk 8 above)
Comment (MikeRyanDev) [reply README.md]: Changed, thanks 👍

File: README.md (hunk 4 above)
Comment (brandonroberts) [reply README.md:77]: Doesn't it work with the `--aot` flag?

Comment (MikeRyanDev) [reply README.md:77]: Complains about not knowing the module for the `FocusTrap` component which seems to be an unregistered component published in @angular/material.

Comment (brandonroberts) [reply README.md:77]: Oh ok

Comment (brandonroberts) [reply README.md:77]: The Focus Trap issue has been fixed in the latest version of Material

Comment (MikeRyanDev) [reply README.md:77]: Ah, then yes the example-app along with the linked angular-webpack2-starter will both be sources for AOT examples.

File: README.md (hunk 8 above)
Comment (btroncone) [reply README.md]: Haha oops! Yes. 👍

**Pull request merged**